service/controller/v14/adapter/guest_outputs.go

```diff
@@ -9,68 +9,70 @@ import (
 //     github.com/giantswarm/aws-operator/service/controller/v14/templates/cloudformation/guest/outputs.go
 //
 
-type outputsAdapter struct {
-	Master        outputsAdapterMaster
-	Worker        outputsAdapterWorker
-	VersionBundle outputsAdapterVersionBundle
+type guestOutputsAdapter struct {
+	Master         guestOutputsAdapterMaster
+	Worker         guestOutputsAdapterWorker
+	Route53Enabled bool
+	VersionBundle  guestOutputsAdapterVersionBundle
 }
 
-type outputsAdapterMaster struct {
+func (a *guestOutputsAdapter) Adapt(config Config) error {
```

Comment:
Why did the method move? IMO the types first and then the methods is correct. Now it is mixed and we have types above and below.